WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadService.java

```diff
@@ -477,6 +518,14 @@ public void onMediaUploaded(OnMediaUploaded event) {
     @SuppressWarnings("unused")
     @Subscribe(threadMode = ThreadMode.MAIN, priority = 7)
     public void onPostUploaded(OnPostUploaded event) {
+        if (event.isError()) {
+            if (event.error != null && event.error != null) {
```

Comment:
Looks like you updated a similar one in `OnMediaUploaded` but missed the triple check in this method @mzorz.